apps/marketplace/pages/LoginPage.js

```diff
@@ -32,15 +32,15 @@ export class LoginPageComponent extends BaseForm {
   }
 
   render() {
-    const { model, loggedIn, handleSubmit, currentlySending } = this.props
-    const { from } = this.props.location.state || { from: { pathname: '/' } }
+    const { model, loggedIn, handleSubmit, currentlySending, userType } = this.props
+    const initialPage = userType === 'buyer' ? '/2/buyer-dashboard' : '/2/seller-dashboard'
```

Comment:
I'm not sure we can merge this - otherwise if anyone does losing via `/2/login` they'll end up on a hardcoded dashboard

Reply:
Happy to leave it in the branch